pyproject.toml

```diff
@@ -116,7 +116,8 @@ Repository = "https://github.com/collective/icalendar/"
 source_archive = "https://github.com/collective/icalendar/archive/{commit_hash}.zip"
 Issues = "https://github.com/collective/icalendar/issues"
 Documentation = "https://icalendar.readthedocs.io/"
-Changelog = "https://icalendar.readthedocs.io/en/latest/changelog.html"
+Changelog = "https://icalendar.readthedocs.io/en/stable/changelog.html"
```

Comment:
The alpha releases do not turn up here although they are released on PyPI. But indeed, it needs correcting. Would you point is to latest?